src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseEditPageUiTest.java

```diff
@@ -462,9 +462,9 @@ private void testDeleteCourseAction() {
         assertNotNull(BackDoor.getCourseAsJson(courseId));
         
         ______TS("delete course then proceed");
-        
+
         InstructorCoursesPage coursePage = courseEditPage.clickDeleteCourseLinkAndConfirm();
-        coursePage.verifyContains("Add New Course");
+        assertTrue(coursePage.getStatus().contains(String.format(Const.StatusMessages.COURSE_DELETED, courseId)));
```

Comment:
I see, ok. There's no need to involve another status message here so I think `.contains` should do good enough.